src/Utils.ts

```diff
@@ -95,6 +95,7 @@ export const bindModelHandlers = (ctx: IEditor, editor: any) => {
   ctx.$watch('value', (val: string, prevVal: string) => {
     if (editor && typeof val === 'string' && val !== currentContent && val !== prevVal) {
       editor.setContent(val);
+      currentContent = editor.getContent();
```

Comment:
You should be able to set `currentContent = val` instead of having to get the editor content as we are setting the content on the line above.